src/components/photoGallery/PhotoGallery.scss

```
@@ -0,0 +1,14 @@
+.imageGallery {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: space-around;
+    gap: 10px;
+}
+
+.spaceImage {
```

Kebab case
```suggestion
.space-image {
```

Remember to change your React code too!